File: Basic Programming/Dimik oj/Python3/প্রোগ্রামিং সমস্যা 35 - [৫২ সমস্যা বই] বৃত্তের বাইরে.py
#                      Name : Arif Khan
#                      Judge: Dimik Oj
#                      University: Primeasia University
#                      problem: প্রোগ্রামিং সমস্যা 35 - [৫২ সমস্যা বই] বৃত্তের বাইরে
#                      Difficulty: Easy
#                      Time Complexity: 0.3301 সেকেন্ড
#                      Problem Link: https://dimikoj.com/problems/35/out-of-the-circle-by
#

import math
if __name__ == '__main__':
  t = int(input())
  for j in range(t):
    x1,y1 = map(float, input().split())
    r = float(input())
    x2,y2 = map(float, input().split())
    dis = math.sqrt(pow(x2-x1,2)+pow(y2-y1,2))
    print("The point is not inside the circle" if dis > r else "The point is inside the circle")

File: Basic Programming/Dimik oj/C++/প্রোগ্রামিং সমস্যা 35 - [৫২ সমস্যা বই] বৃত্তের বাইরে.cpp
/*                    Name : Arif Khan
                      Judge: Dimik Oj
                      University: Primeasia University
                      problem: প্রোগ্রামিং সমস্যা 35 - [৫২ সমস্যা বই] বৃত্তের বাইরে
                      Difficulty: Easy
                      Time Complexity: 0.2277 সেকেন্ড
                      Problem Link: https://dimikoj.com/problems/35/out-of-the-circle-by
*/

#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
	cin >> n;
	while(n--) {
	    double  x1, y1, x2, y2, r, m;
		cin >> x1 >> y1 >> r >> x2 >> y2;
	    m = ((x2-x1)*(x2-x1))  +  ((y2-y1)*(y2-y1));
		double dis = sqrt(m);
		if(dis > r) {
		    cout << "The point is not inside the circle" << endl;
		}else{
		    cout << "The point is inside the circle" << endl;
		}
	}
	return 0;
}